stage0/src/fw_cfg.rs

```diff
@@ -130,6 +140,11 @@ impl FwCfg {
         })
     }
 
+    pub fn find(&mut self, name: &CStr) -> Option<DirEntry> {
```

Comment:
Why do you need `CStr` ? Are you calling into C somewhere?

Reply:
Indirectly -- the `fw_cfg` file names are null-terminated, like C strings are. Thus, it makes sense to use the Rust `CStr` for them.